pandas/core/generic.py

```diff
@@ -3620,6 +3619,10 @@ def astype(self, dtype, copy=True, errors='raise', **kwargs):
             the same type. Alternatively, use {col: dtype, ...}, where col is a
             column label and dtype is a numpy.dtype or Python type to cast one
             or more of the DataFrame's columns to column-specific types.
+        copy : bool, default True.
+            Return a copy when ``copy=True`` (be very careful setting
+            ``copy=False`` as changes to values then may propagate to other
+            pandas objects).
```

Comment:
Hmm, I consciously made it clearer but not-too-specific, as I don't really know how ``copy`` does its thing, and I doubt this is used so often.

I suggest to leave it as it is. Is that ok?